mythril/laser/ethereum/util.py

```diff
@@ -10,6 +10,10 @@
 TT255 = 2 ** 255
 
 
+class ConcreteIntException(AttributeError):
```

Comment:
Why did you add this specific exception here? From the name its not immediately clear what kind of situation it is meant for and I think in this case that a Type error is more adequate for the situation

Reply:
There are 2 exceptions possible here, so I thought lets return this, or should i return TypeError when a ValueError is raised?

Reply:
I'd like one of two situations:
Either we use a TypeError, because it describes the problem that happens. Namely we assume an element has a certain type, while it is not.
Or we do use a specific exception, but then I think it should have a name that describes what the problem is. ```ConcreteIntException``` isn't immediately clear in that aspect (at least to me)